app/controllers/hubstats/application_controller.rb

```diff
@@ -5,9 +5,14 @@ class ApplicationController < ApplicationController
 
     private
     def set_time
-      cookies[:hubstats_index] ||= 2
-      @start_date = DATE_RANGE_ARRAY[cookies[:hubstats_index].to_i][:date].ago.to_date
-      @end_date = Time.now
+      if cookies[:hubstats_index] == "null~~null" || cookies[:hubstats_index] == nil
```

Comment:
If someone presses Apply without any dates selected, we get:
![screen shot 2015-06-24 at 1 53 43 pm](https://cloud.githubusercontent.com/assets/7562793/8338759/9622b284-1a78-11e5-9cf1-1aa4fd55c29d.png)

as an index. So we need to check if this is the case, and then if so, just return the default date range.